compiler/p/codegen/OMRTreeEvaluator.cpp

```diff
@@ -4577,6 +4577,7 @@ TR::Register *OMR::Power::TreeEvaluator::arraycopyEvaluator(TR::Node *node, TR::
 
    static bool disableLEArrayCopyHelper  = (feGetEnv("TR_disableLEArrayCopyHelper") != NULL);
    static bool disableVSXArrayCopyInlining = (feGetEnv("TR_disableVSXArrayCopyInlining") != NULL);
+   disableVSXArrayCopyInlining = true; // Disabling due to a performance regression
```

Comment:
If you're going to leave the code in place and just disable it perhaps

```
static bool disableVSXArrayCopyInlining = (feGetEnv("TR_enableVSXArrayCopyInlining") == NULL);
```

would be better.